src/excited_workflow/source_datasets/land_cover.py

```diff
@@ -32,19 +130,21 @@ def load(
         """
         self.validate_variables(variables)
 
-        files = list(self.get_path().glob("*.nc"))
+        if target_grid is None:
+            msg = "target_grid is not optional for loading landcover data."
+            raise ValueError(msg)
+
+        path = self.get_path() / "preprocessed"
```

Comment:
```suggestion
        preprocessed_dir = self.get_path() / "preprocessed"
```
This is a more clear name I think.